population_queries.sqlite

```diff
@@ -0,0 +1,37 @@
+-- This is the first query:
+SELECT DISTINCT year from population_years;
+
+-- Add your additional queries below:
+--The largest population size for Gabon in this dataset
+SELECT DISTINCT population
+FROM population_years
+WHERE country = "Gabon"
+ORDER BY population DESC
+LIMIT 1;
+
+--10 lowest population countries in 2005
+SELECT *
+FROM population_years
+WHERE year = 2005
+ORDER BY population ASC
+LIMIT 10;
+
+--All the distinct countries with a population of over 100 million in the year 2010
+SELECT DISTINCT country, population
+FROM population_years
+WHERE population > 100.0
+AND year = 2010
+ORDER BY population DESC;
+
+--How many countries in this dataset have the word "Islands" in their name
+SELECT COUNT(*) FROM (SELECT DISTINCT country
+FROM population_years
+WHERE country like '%Islands%');
+
+
+--The difference in population between 2000 and 2010 in Indonesia
+SELECT MAX(population) - MIN(population) AS populationDIFF FROM
+population_years
+WHERE country = 'Indonesia';
+
```

Comment:
To get the population between 2000 & 2010 I would add `AND (year >= 2000 and year <= 2010)`

Reply:
I'll make the update.